app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import NotVaccinatedError, OutdatedVaccineError
+from app.errors import NotWearingMaskError
```

Comment:
Code Style: According to the checklist, you should use parentheses when importing multiple classes/modules from the same module. It would be better to write it like this:

python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```